vbench/src/vbench/vbench/request_scheduler.cpp

```diff
@@ -1,11 +1,13 @@
 // Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 
 #include "request_scheduler.h"
-
 #include <vbench/core/timer.h>
 
 namespace vbench {
 
+VESPA_THREAD_STACK_TAG(vbench_request_scheduler_executor);
```

Comment:
Consider _thread instead of _executor as this is used for vespalib::Thread.

Reply:
Fixed